Druid/DRBasicCogInterpreterPrimitivesWithoutParameters.class.st

```diff
@@ -0,0 +1,26 @@
+Class {
+	#name : #DRBasicCogInterpreterPrimitivesWithoutParameters,
+	#superclass : #CogVMSimulatorLSB,
```

Comment:
Same here

Reply:
It is referenced now in #interpreterClassUnaryPrimitives